packages/react-pagination/package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "@wpmudev/react-pagination",
+  "version": "0.0.0",
+  "description": "WPMU DEV Shared UI React Pagination Component",
+  "keywords": [],
+  "author": "WPMU DEV (https://wpmudev.com/)",
+  "contributors": [
+    {
+      "name": "Leighton Sapir",
+      "email": "2328848+iamleigh@users.noreply.github.com",
+      "url": "https://iamleigh.com/"
+    }
```

Comment:
@Gowtham369 don't forget to add yourself to the contributors' list. Use your Incsub email or the secure one from Github (similar to mine).